### DISCUSSION TITLE
Downloader isnt working

### DISCUSSION THREAD

**Discussion by woducku** [tonymushah/special-eureka#36] [Mangadex Special-Eureka Q/A]
![image](https://user-images.githubusercontent.com/57977673/228339016-b0095c65-e87d-4e7e-af62-d2aef3547ebf.png)
also do i have to enable any setting?
![image](https://user-images.githubusercontent.com/57977673/228339397-d68e80e9-c2ee-41b3-ab7d-b15cd6604528.png)

**Comment by tonymushah** [comment]
You should enable the offline server first to download a chapter.
Also thanks for reporting.

**Comment by woducku** [comment]
### Question
How do u make the server online?

### Bug
1. English as the language is selected, it is still showing other languages content. (also it's not clear which language it's indicating. original source language or translation language.)
![image](https://user-images.githubusercontent.com/57977673/228804753-da998f0f-b817-4877-9ac1-9e25b35c9bb7.png)
![image](https://user-images.githubusercontent.com/57977673/228804767-ca2025b5-6e52-485c-8b0f-171a9de6c479.png)

### Suggestion/Request/Rant
1. the load time is very long and stops loading after a minute or so. 
2. also a dark mode feature will be appreciated 
3. maybe make the links black/white (depending on the theme) instead of hyperlink blue.
![image](https://user-images.githubusercontent.com/57977673/228804882-5c169f31-c74f-444d-9aec-d19db2c3e76f.png)

**Comment by tonymushah** [comment]
`
How do u make the server online?
`
Just click on the server icon and it should turn to green 

like so 

![Capture d’écran 2023-03-30 201712](https://user-images.githubusercontent.com/95529016/228914856-311c16d6-8bee-4bdf-ab89-ed7e613e9464.png)

**Comment by tonymushah** [comment]
- For the loading time, i'm already found some solutions which will decrease the mangadex home page loading time. 
- For the languages, it's for translated language only (i will precise it on the next realease)
- For the UI, i will progressively change them 

If you find something, please tell me

**Comment by woducku** [comment]
> `How do u make the server online?` Just click on the server icon and it should turn to green
> 
> like so
> 
> ![Capture d’écran 2023-03-30 201712](https://user-images.githubusercontent.com/95529016/228914856-311c16d6-8bee-4bdf-ab89-ed7e613e9464.png)

![image](https://user-images.githubusercontent.com/57977673/228981511-ea5e932e-2ca6-4ea5-ae6a-1b15e6169fa1.png)
doesnt work.

**Comment by tonymushah** [comment]
Okay, do you use Linux or Windows?
or do you use the dev build?

This error happens because the app doesn't have the permission to host the offline server server locally.
(Yes, i know hosting a local server on PC is not the best approach for the downloader system)

I'll check this error out.

Thanks for reporting.
